app/test_main.py

```diff
@@ -1,3 +1,31 @@
+import pytest
+
+from typing import Any
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        pytest.param(-1, -1, [0, 0], id="test when animals 'age' < 0"),
+        pytest.param(0, 0, [0, 0], id="test when animals 'age' == 0"),
+        pytest.param(23, 23, [1, 1], id="test must return [1, 1] when age<24"),
+        pytest.param(36, 36, [5, 4], id="test should return different age")
```

Comment:
add more cases here, what about big nums ? what about other cases, like this 
![image](https://github.com/mate-academy/py-cat-and-dog-years/assets/103201010/1c444dba-b3e1-4067-8e93-09cdd39c4323)
![image](https://github.com/mate-academy/py-cat-and-dog-years/assets/103201010/2c965b47-13af-4faf-8f36-17d07839196a)